HelpSource/scdoc.css

```diff
@@ -611,17 +621,18 @@ ul.tree li {
 }
 
 ul.tree li:last-child {
- background: #fff url(images/lastnode.png) no-repeat;
+ background: var(--sc-bg-color) url(images/lastnode.png) no-repeat;
 }
 
+/* is this still used? */
 :target {
-    border-bottom: 6px solid rgba(255,255,0,0.4);
+    border-bottom: 6px solid var(--sc-bg-warning);
     padding-top:20px;
     margin-top:-20px;
 }
 
 a.undoc {
-    color: #88b;
+    color: var(--sc-text-color);
```

Comment:
This would be indeed a good idea but CSS does not allow for this (at least currently, see https://caniuse.com/mdn-css_types_color_color-mix )

I switched to `--sc-text-color-lighter` instead.

Reply:
How about:
```css
a.undoc {
   color: var(--sc-link-color);
   opacity: 50%;
}
```